README.md

```diff
@@ -1,4 +1,8 @@
-<img width="50%" alt="Box Elements" src="https://repository-images.githubusercontent.com/95743138/c161b500-021b-11ea-8bf9-3aa8776acdec" />
+<p align="center">
+  <img width="50%" alt="Box Elements logo" src="https://repository-images.githubusercontent.com/95743138/c161b500-021b-11ea-8bf9-3aa8776acdec" />
+  <img width="50%" alt="Box Developer logo" src="https://github.com/box/sdks/blob/master/images/box-dev-logo.png" />
```

Comment:
[nit] I didn't realize you could link to github.com here. I don't think this will render the image properly in vanilla html. It looks like Github actually makes two network requests for this image (which doubles the image latency).
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/34158284/226722265-7fc3c2a2-f9fc-4345-a161-c8aa15cdcf4b.png">

Maybe we should replace it with the raw link or permalink so it points directly to Github's CDN?
- https://raw.githubusercontent.com/box/sdks/master/images/box-dev-logo.png
- https://raw.githubusercontent.com/box/sdks/f55569c6f023a5dfddad96729b87a259c9bf4113/images/box-dev-logo.png

Reply:
oh good point

Reply:
I think permalink is better so we can avoid link rot (if `box-dev-logo.png` is moved/renamed/removed from `master` or we rename `master` to `main`, for example). The small tradeoff is just that we'd have to update the link here if the logo changes.